### PR TITLE
Update to be inclusive

### PR DESCRIPTION
**Pull request by alikoneko**
I'm named Ali. I struggle with body image, and have my entire life. Can we at least be nice and refer to a pet and maybe not body image issues.

### CODE REVIEW

File: ternary_operators.rst [hunk 1]
```diff
@@ -38,10 +38,10 @@ is some sample code:
...
 .. code:: python
 
-    fat = True
-    fitness = ("skinny", "fat")[fat]
-    print("Ali is ", fitness)
-    # Output: Ali is fat
+    nice = True
+    personality = ("mean", "nice")[nice]
+    print("The cat is ", personality)
+    # Output: The cat is mean
```
Comment (yasoob) [inline ternary_operators.rst]: Can you change the output to nice?

**Comment by yasoob** [comment]
Hi @alikoneko I am so sorry for this. I have added a comment to your PR. I will merge this PR as soon as that issue is fixed. Have a good day!

**Comment by alikoneko** [comment]
Thank you for being understanding. I have battled body image my entire
life, and this ruined my night a couple days ago. You have no idea how much
this means to me.

**Comment by yasoob** [comment]
Thank you very much @alikoneko

**Pull request merged**